image_topic: '/spinnaker_ros_driver_node/cam_fm_01/image_raw'
output_path: '/home/bzeren/Desktop/bag-data-2/'
delay_time: 0.7